Firestore/core/src/firebase/firestore/timestamp.cc

```diff
@@ -31,10 +31,6 @@ namespace firebase {
 
 namespace {
 
-// We pass it by value without std::move.
-static_assert(std::is_trivially_copyable<Timestamp>::value,
```

Comment:
This currently has problems building internally. It seemed to me like doing complicated workarounds isn't justified for this case (after all, the data layout of `Timestamp` is very unlikely to change). Happy to provide more details offline.